Glass-Mint-Classic/files/Glass Mint Classic/cinnamon/cinnamon.css

```diff
@@ -120,6 +120,26 @@ StScrollBar StButton#vhandle:hover
 	font-size: 13px;
 }
 
+.menu {
+/* ('RU') Сама плоскость со всплывающим меню. */
+	color: rgb(230,230,230);
+    -boxpointer-gap: 2px;
+    -arrow-border-color: transparent;
+    -arrow-background-color: transparent;
+    -arrow-border-width: 0px;
+    -arrow-border-radius: 0px;
+    border-radius: 8px;
+    background-gradient-direction: vertical;
+    background-gradient-start: rgba(46,45,42,0.8);
+    background-gradient-end: rgba(46,45,42,0.8); /* это цвет меню */
```

Comment:
How come you're still theming things with the old menu properties? Like -boxpointer-gap. Was it just an oversight here? Or maybe copy/paste error?